dev-tools/scripts/releaseWizard.yaml

```diff
@@ -1067,6 +1067,70 @@ groups:
         cmd: svn -m "Add docs, changes and javadocs for Solr {{ release_version }}"  import {{ git_checkout_folder }}/solr/build/docs  https://svn.apache.org/repos/infra/sites/solr/docs/{{ version }}
         logfile: add-docs-solr.log
         comment: Add docs for Solr
+  - !Todo
+    id: build_final_ref_guide
+    title: Build the ref guide
+    links:
+      - https://solr.apache.org/guide/{{ release_version_refguide }}
+    commands: !Commands
+      root_folder: '{{ git_checkout_folder }}'
+      commands_text: Build the Solr Ref Guide
+      commands:
+        - !Command
+          cmd: "{{ gradle_cmd }} solr:solr-ref-guide:clean"
+          comment: Clean the Ref Guide build
+        - !Command
+          cmd: "{{ gradle_cmd }} solr:solr-ref-guide:buildSite -PsolrGuideDraft=false"
+          comment: Build Ref Guide
+  - !Todo
+    id: upload_final_ref_guide
+    title: Upload the ref guide
+    links:
+      - https://solr.apache.org/guide/{{ release_version_refguide }}
+    depends:
+      - build_final_ref_guide
+    types:
+      - major
+      - minor
```

Comment:
the bugfix releases are handled in the next section 🙂 

It unfortunately has to be separated, because it's adding new files vs. uploading just the changes.